content/css/concepts/animations/terms/animation-delay/animation-delay.md

```diff
@@ -87,3 +87,7 @@ div {
   }
 }
 ```
+
+![Example 1 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example1.gif)
+
+![Example 2 animation](https://raw.githubusercontent.com/Codecademy/docs/main/media/animation-delay-example2.gif)
```

Comment:
@Kadif-Ravik can be specific in the tags what the animation will do. Like Animations example with delay of 2 seconds.